### ISSUE TITLE
Create Primary Rounded button blue

### ISSUE THREAD

**Issue by RavindraP04** [Clueless-Community/seamless-ui#1609]
### Here is a reference to the component. 
![image](https://user-images.githubusercontent.com/98400348/216666162-b803e424-04e0-42c9-a889-bf201c3962d0.png)
--
### After building the component please raise a PR with a screenshot of the component and add the component in `Button/src/Primary/text-button-rounded.html` by replacing the existing code and rename the file to `text-button-rounded-blue.html`. 

So, In short you need to do:-
- Make the Component
- Replace the code of `Button/src/Primary/text-button-rounded.html` with your code
- Rename `text-button-rounded.html` to `text-button-rounded-blue.html`
  
If you need to use any icon please use it from [Hero Icons](https://heroicons.com/)

Good luck.

**Comment by rt-001** [comment]
i want to work on this issue

**Comment by SakshiShah29** [comment]
I would like to work on it.
